1-Collect/Get-RessourcesFromAM.ps1

```diff
@@ -0,0 +1,138 @@
+﻿<#.SYNOPSIS
+    Parses Azure Migrate assessment data from an Excel file and extracts virtual machine and disk SKUs.
+.DESCRIPTION
+    This script retrieves and processes Azure resource data from the output file generated by Azure Migrate, specifically focusing on virtual machine and disk SKUs.
+    It outputs the results in the same structured JSON format as Get-AzureServices.ps1, which can then be used for finding appropriate Azure regions where these resources can be deployed.
+.PARAMETER filePath
+    The path to the Excel file containing the Azure Migrate assessment data.
+
+.PARAMETER outputFile
+    The name of the output file where the results will be exported. Default is "resources.json".
+
+.EXAMPLE
+    PS C:\> .\Get-RessourcesFromAM.ps1 -filePath "C:\path\to\Assessment.xlsx" -outputFile "C:\path\to\summary.json"
+    Runs the script with the specified Excel file and outputs the results to the specified JSON file.
+
+.OUTPUTS
+    JSON file containing the queried resource data and extracted properties.
+
+.NOTES
+    - Requires Azure PowerShell module to be installed and authenticated.
+    - Requires ImportExcel module to be installed for reading Excel files.
+    - The script assumes the Excel file has specific worksheets named 'All_Assessed_Machines' and 'All_Assessed_Disks'.
+
+#>
+
+param(
+    [Parameter(Mandatory=$true)] [string]$filePath,
+    [Parameter(Mandatory = $false)] [string]$outputFile = ".\summary.json" # Json file to export the results to
+)
+
+# Check if the file exists
+if (-Not (Test-Path -Path $filePath)) {
+    Write-Output "File not found: $filePath"
+    exit
+}
+# Ensure the ImportExcel module is installed
```

Comment:
I would remove line 37-50 and instead put in the requirement for the importexcel module in the synopsis in the beginning